src/features/iam/iamState.ts

```diff
@@ -19,6 +19,20 @@ export class IamState {
     return (await this.fetchProjects()).find((p) => p.projectId === projectId)
   }
 
+  async getActiveProject(): Promise<ProjectDto> {
```

Comment:
Since this throws an error, let's call it `requireActiveProject` to be consistent